src/ngraph/pass/reshape_sinking.cpp

```diff
@@ -106,9 +108,9 @@ static void insert_reshape(shared_ptr<Node> target, shared_ptr<Node> reshape, si
 
 static void delete_reshape(shared_ptr<Node> reshape)
 {
-    NGRAPH_DEBUG << "Removing reshape " << reshape->get_name();
     if (!reshape->get_users().empty())
     {
+        NGRAPH_DEBUG << "Removing reshape " << reshape->get_name();
```

Comment:
`NGRAPH_DEBUG << "Removing shape " << reshape;`

Reply:
nice! i recall recent PRs in master has some nice print handling, will convert to this.